01-hello-git/README.md

```diff
@@ -66,194 +67,191 @@ or receiving changes from other repositories.
 
 ## Exercises
 
-### Creating a Fork
 
-A _fork_ is a personal copy of a repository hosted by a provider such as
-[GitHub](https://github.com). Create a fork and clone it.
+### Installing Git
+Before you start using Git, you have to make it available on your computer.
```

Comment:
I suggest something like:

## Prerequisites

* [Git](https://git-scm.com/downloads)

(The directions should be straightforward for users of the various operating systems, as the installation methods provided are what users are already used to when installing software - e.g. Windows provides an .exe installer. We may want to add a comment that suggests using `nano` as the editor, since it's not the default, but otherwise the defaults seem fine)

Reply:
The only reason that I have mentioned the installing procedure is that just one single document (say README.md) can address the new team members' needs. As you know there are a large number of links and documents that explain the prerequisites and I think just providing a link in README.md file is not useful. However,  please let me know if you still believe that we do not need this part.